doc/tags/for.rst

```diff
@@ -55,9 +55,9 @@ The ``loop`` variable
 
 Inside of a ``for`` loop block you can access some special variables:
```

Comment:
Describing `loop.changed` as a variable looks weird to me, as `changed` is not a getter but a method requiring an argument.

Anyway, from a strict point of view, there is only 1 special variable: `loop`. What this table describes is the API of this loop variable.